docs/01-get-started/your-first-route.mdx

```diff
@@ -6,12 +6,48 @@ sidebar_position: 3
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+# Routes
 
-Add instructions here:
+Starting v3.0.0, Front-Commerce uses Remix's
+[API Routes](https://remix.run/docs/en/main/guides/api-routes) to handle all the
+routing of you application. All routes are located under `app/routes` folder.
```

Comment:
**issue:** this is the v3 doc. No need to mention how it was before or that it is something new in v3.

**suggestion:** 
```suggestion
Front-Commerce uses Remix. Remix uses
[API Routes](https://remix.run/docs/en/main/guides/api-routes) to handle all the
routing of you application, and display the correct page when a user navigates to a URL. All routes are located under `app/routes` folder.
```